src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ReferenceTransactionNumber.java

```diff
@@ -0,0 +1,55 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+@Getter @Setter
+@RequiredArgsConstructor @NoArgsConstructor
```

Comment:
A bit a picky one, but do you think that for consistency we should separate these by new lines as it's done in the different classes (for example in MessageTrailer class, the @Getter, @Setter and @RequiredArgsConstructor are separated by a new line). If not, that's fine

Reply:
Agreed and fixed in 42245a5